packages/suite/src/components/suite/FormattedCryptoAmount/index.tsx

```diff
@@ -18,7 +22,7 @@ const FormattedCryptoAmount = ({ value, symbol, disableHiddenPlaceholder, classN
     const content = (
         <>
             <Value>{value}</Value>
-            {symbol && <Symbol>{` ${symbol.toUpperCase()}`}</Symbol>}
+            {symbol && <Symbol>{symbol}</Symbol>}
```

Comment:
I liked the previous version more as adding space vs hardcoded 4px margin is not the same. Extreme example: space has different width with font-size: 12px than with font-size:64px, I would like to keep this component reusable for all font sizes.

Reply:
solved